acceptance/launcher_test.go

```diff
@@ -353,3 +363,14 @@ func assertOutput(t *testing.T, cmd *exec.Cmd, expected ...string) {
 		}
 	}
 }
+
+func copyPosixFixture(t *testing.T) {
```

Comment:
Is this copying things from `posix/` because the `Dockerfile` is re-building the go binaries? I wonder if building a known named image to use would be more discoverable. Like `buidpacksio/exechelper` and both `Dockerfile` would change to `FROM buidpacksio/exechelper as builder`. This obviously assumes the dockerfile has been built.

Reply:
Related discussion: https://github.com/buildpacks/lifecycle/pull/534#discussion_r582321165